mithril-stm/src/schnorr_signature/mod.rs

```diff
@@ -0,0 +1,173 @@
+// TODO: Remove
+#![allow(dead_code)]
+
+mod signature;
+mod signing_key;
+pub(super) mod utils;
```

Comment:
What's the purpose of `pub(super)` when you have `pub use utils::*;` below?
```suggestion
mod utils;
```